src/ComposerRequireChecker/FileLocator/LocateComposerPackageSourceFiles.php

```diff
@@ -6,11 +6,8 @@
 
 final class LocateComposerPackageSourceFiles
 {
-    public function __invoke(string $composerJsonPath): Generator
+    public function __invoke(array $composerData, string $packageDir): Generator
```

Comment:
Please document this `array` parameter: I know this is mostly `mixed[]`, but that would still be needed

Reply:
I've added `@param mixed[] $composerData The contents of composer.json for a package` now